### PR TITLE
Add batch support to client

### PR DESCRIPTION
**Pull request by eivinhb**
Digipost has just released support for batches in Api. This PR add the usage of this to the client.

image

### CODE REVIEW

File: src/main/resources/xsd/api_v8.xsd [hunk 1]
```diff
@@ -1279,4 +1280,58 @@
...
+                <xsd:annotation>
+                    <xsd:documentation>The batch has received documents and can receive more documents.</xsd:documentation>
+                    <xsd:appinfo>
+                        <jaxb:typesafeEnumMember name="NOT_COMMITTED" />
+                    </xsd:appinfo>
+                </xsd:annotation>
+            </xsd:enumeration>
+            <xsd:enumeration value="COMMITTED">
+                <xsd:annotation>
+                    <xsd:documentation>The batch has commitet and documents are being processed. The batch can not receive any more documents.</xsd:documentation>
```
Comment (draperunner) [inline src/main/resources/xsd/api_v8.xsd:1320]: `has commitet` -> `has been committed`?

**Review by draperunner — APPROVED** [review]

File: src/main/java/no/digipost/api/client/representations/batch/Batch.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+import static java.util.Optional.ofNullable;
+import static no.digipost.api.client.representations.Relation.COMPLETE_BATCH;
+import static no.digipost.api.client.representations.Relation.SELF_DELETE;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "batch", propOrder = {
+        "uuid",
+        "status",
+        "antallDigipost",
```
Comment (post-svejk) [inline src/main/java/no/digipost/api/client/representations/batch/Batch.java:39]: Denne og neste, burde kanskje ha engelske navn?

File: src/main/java/no/digipost/api/client/representations/batch/Batch.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+})
+@XmlRootElement(name = "batch")
+public class Batch extends Representation {
+
+    @XmlElement(required = true)
+    protected String uuid;
+    @XmlSchemaType(name = "string")
+    protected BatchStatus status;
+    @XmlElement(name = "antall_digipost")
+    protected Integer antallDigipost;
```
Comment (post-svejk) [inline src/main/java/no/digipost/api/client/representations/batch/Batch.java:51]: Engelsk navn?

**Review by post-svejk — APPROVED** [review]
Looks good. Concise and clear docs.

**Pull request merged**